frame/base-fee/src/lib.rs

```diff
@@ -27,6 +27,7 @@ pub mod pallet {
 
 	pub trait BaseFeeThreshold {
 		fn lower() -> Permill;
+		fn mid() -> Permill;
```

Comment:
`mid` is not a great name IMO. It implies middle or 50% or something. Maybe `target` or `ideal` is better?

Kind of just bike-shedding though. This change looks good to me.